src/ngx_http_lua_directive.c

```diff
@@ -1304,11 +1298,11 @@ ngx_http_lua_gen_chunk_name(ngx_conf_t *cf, const char *tag, size_t tag_len,
 
 found:
 
-    ngx_snprintf(out, len, "=%*s(%*s:%d)%Z",
+    p = ngx_snprintf(out, len, "=%*s(%*s:%d)%Z",
```

Comment:
style: the below arguments are not aligned with `(out` anymore

Reply:
My bad. Forgot to run `ngx-releng` :) Already pushed a fix.